ray-curator/ray_curator/stages/text/embedders/utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cudf
+import cupy as cp
+import pylibcudf as plc
+
+
+def create_list_series_from_1d_or_2d_ar(ar: cp.ndarray, index: cudf.Index) -> cudf.Series:
```

Comment:
Could you add a comment with the appropriate link, saying that this function comes from CrossFit?